chat-plugins/uno.js

```diff
@@ -0,0 +1,576 @@
+'use strict';
+
+const maxTime = 30; // seconds
+
+const rgb_Gradients = {
+	"Green": "rgba(0, 122, 0, 1), rgba(0, 122, 0, 0.65)",
+	"Yellow": "rgba(255, 255, 0, 1), rgba(255, 255, 0, 0.65)",
+	"Blue": "rgba(0, 0, 255, 1), rgba(0, 0, 255, 0.65)",
+	"Red": "rgba(255, 0, 0, 1), rgba(255, 0, 0, 0.65)",
+	"Black": "rgba(0, 0, 0, 1), rgba(0, 0, 0, 0.65)",
+};
+
+const text_shadow = "text-shadow: 1px 0px black, -1px 0px black, 0px -1px black, 0px 1px black, 3px -3px black;";
```

Comment:
ok updated, sorry for the wait, choir and other "educational" things happened first